src/Microsoft.AspNet.Mvc.Razor/IInitializeTagHelperOfT.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Razor.Runtime.TagHelpers;
+
+namespace Microsoft.AspNet.Mvc.Razor
+{
+    /// <summary>
+    /// Initializes an <see cref="ITagHelper"/> before it's executed.
+    /// </summary>
+    /// <typeparam name="TTagHelper">The <see cref="ITagHelper"/> type.</typeparam>
+    public interface IInitializeTagHelper<TTagHelper>
```

Comment:
Out of curiosity - why not call this `ITagHelperInitializer`. It follows the naming convention used by other things - `IControllerActivator`, `ITagHelperActivator` etc